src/RequestLoginNameDialog.cpp

```diff
@@ -0,0 +1,37 @@
+#include "RequestLoginNameDialog.h"
+#include "ui_RequestLoginNameDialog.h"
+#include <QPushButton>
+
+RequestLoginNameDialog::RequestLoginNameDialog(const QString &service, QWidget *parent) :
+    QDialog(parent),
+    ui(new Ui::RequestLoginNameDialog)
+{
+    ui->setupUi(this);
+    setFixedSize(size());
+    setWindowTitle(tr("Request Login Name"));
+    ui->loginLabel->setTextFormat(Qt::RichText);
+    ui->loginLabel->setText(tr("Login name for ") + "<b>" + service + "</b>:");
+    const QString informationText = "<h2><b>"
+                                  + tr("A credential without a login has been detected.")
```

Comment:
Change the text to:

`<h2><b>A credential without a login has been detected.</b></h2><br><br>You have the possibility to include a login before storing the credential in the device. If you don't need a login, you can ignore the request.`

Also, could you include an `ignore` button in the buttonbox